.circleci/config.yml

```diff
@@ -330,3 +330,5 @@ workflows:
             branches:
               only:
                 - series/2.x
+            tags:
```

Comment:
Hmm... the problem is that I usually update the docs with the new version *after* it's published (to avoid the docs being up with a version that doesn't exist yet)

Reply:
Would requiring the `release` step to be run first solve this?

I can add this to the task:

```yaml
requires:
  - release
```

Reply:
That would work, but then what about when we want to update docs without a release?

Reply:
Hmm one way would be to run the `npm run docs:build` locally and merge it into main, but that might be a bit too cumbersome.

Reply:
I suggest for now, we put docs in a separate PR, and if we find a better solution for publishing docs we do it in a dedicated PR?

Reply:
What about if we made the `microsite` workflow require manual approval? https://circleci.com/docs/configuration-reference/#type

We can "trial" it in a separate PR to see if we like it / works

Reply:
I think the more fundamental issue with pushing "future" versions docs into the main branch is that until we release a new version, we can't update the docs anymore for other purposes. Any other change will have to wait a release. That's why I initially set it up this way, to have the freedom to update it many times. In practice though, we usually update it more often when we change something and need to wait for the release. But the flexibility is nice 🤔

Reply:
I guess an alternative would have to have docs versioning with an "unstable" version as well (like sttp/tapir), but it's work to set it up (not even sure vuepress can do that).